src/main/java/com/example/petbuddybackend/service/chat/ChatService.java

```diff
@@ -74,14 +74,17 @@ public ChatMessageDTO createMessage(
             Role role,
             ZoneId timeZone
     ) {
```

Comment:
z tego co widze to powyzszej metody nigdzie nie uzywamy poza testami

Reply:
Faktycznie. Usunąłem